### DISCUSSION TITLE
Conexión a Power BI

### DISCUSSION THREAD

**Discussion by PedroReus** [dbeaver/dbeaver#14378] [General]
Hello, good afternoon, first of all, an excellent platform initiative in sql language which has facilitated and optimized the way I work saving labor hours.

My requests are:

DBeaver program can connect to power bi to import or export information to be able to make reports to managers or business owners.
Another request that I find that the program can improve is that when extracting information it can be in ".xlsx", "accdb" format.
This will allow greater interaction with other applications and will encourage the use of the system

**Comment by LonwoLonwo** [comment]
Hello @PedroReus 

DBeaver can export tables to XLSX format with Office extension: https://github.com/dbeaver/dbeaver/wiki/Optional-extensions#office-integration

**Comment by PedroReus** [comment]
Thanks for the prompt reply and help.
The office extension installed without problems.

Now my other point is if the installation of the office extension works to connect tables to power bi and if so, how does it work?

**Comment by p91paul** [comment]
What do you mean by connect to power bi? Power BI is not a database. It's either Power BI desktop, which is a database (and other datasources) client, who could even be seen as a DBeaver alternative with a radically different approach to database querying, or Power BI server, which is more or less an archive for reports produced with the desktop application.

**Comment by PedroReus** [comment]
What I am asking is if I can connect dbeaver with power bi desktop and then on power bi desktop perform dashboard

**Comment by p91paul** [comment]
So you want to create a Power BI dashboard based on data you see in DBeaver?
You should probably just connect Power BI to the same database DBeaver is connected to.

**Comment by PedroReus** [comment]
The base to which dbeaver is connected is sql oracle, which I have seen videos, without positive results.
I thought that dbeaver could, within its logic, perform this action.
Do you know how I can solve this situation if I run it directly from sql oracle to power bi desktop

**Comment by p91paul** [comment]
try following this guide https://docs.microsoft.com/en-us/power-bi/connect-data/desktop-connect-oracle-database

**Comment by Paruccone** [comment]
I see this is still open. What the person was (probably) asking for is the ability to connect to Power BI (yes, power bi Desktop is also a DB engine based on vertipaq) without using additional (and paid) components like this: https://www.cdata.com/kb/tech/powerbixmla-odbc-dbeaver.rst
